### DISCUSSION TITLE
New RN version 0.23

### DISCUSSION THREAD

**Discussion by kauly** [tamagui/tamagui#1958] [Q&A]
Is tamagui working with the newer RN version 0.73? Because I'm getting strange errors like these after upgrade:
```
TypeError: _$$_REQUIRE(_dependencyMap[11], (...)tamagui/core").internalWithTheme is not a function (it is undefined), js engine: hermes 
```

**Comment by kauly** [comment]
More context:
![image](https://github.com/tamagui/tamagui/assets/15254623/f0d20343-166d-4823-92e6-af256b0236c6)

**Comment by kauly** [comment]
The error is unrelated with Tamagui, it was about the `react-native-svg-transformer` library. They release a new version compatible with RN 0.73*

**Comment by byteab** [comment]
Thanks for reporting
will fix with this PR https://github.com/tamagui/tamagui/pull/1961

**Reply by kauly** [reply]
Thank YOU
